src/vue/modules/pre-issuance-form/wrappers/asset.js

```diff
@@ -0,0 +1,8 @@
+import safeGet from 'lodash/get'
+
+export class Asset {
```

Comment:
useless record

Reply:
It safely gets and encapsules `owner` property here. Should we just check `a => safeGet(a, 'owner.id') === ownerAccountId` inside mixin?

https://github.com/tokend/web-client/blob/4d1d177e31ce8a8c9bae505aaca8637f1ecb325e/src/vue/modules/pre-issuance-form/mixins/load-owned-assets.mixin.js#L11-L19